newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -267,6 +275,27 @@ export default class ThemableEventsTree extends Component<EventsTreeProps, *> {
     this.onHeightsChanged();
   }
 
+  componentDidUpdate(prevProps: EventsTreeProps) {
+    const {
+      values: { hiddenTutorialHints },
+    } = this.props.preferences;
+    const {
+      values: { hiddenTutorialHints: previousHiddenTutorialHints },
+    } = prevProps.preferences;
+    if (
+      hiddenTutorialHints['intro-event-system'] !==
+      previousHiddenTutorialHints['intro-event-system']
+    ) {
+      this.setState({
+        ...this.state,
+        treeData: this.state.treeData.slice(
```

Comment:
Is there a way to do this filtering but without relying on the fact that this tutorial line is at the end (which might change in the future, and would create an invisible bug)?

Reply:
Yes, it's not ideal but can be a bit more bullet proof by filtering (though you also need to add something to differentiate the tutorial from the rest)

Reply:
I added a key `eventstree-tutorial-node` to the tutorial node (I felt it was more reliable than using "indexInList")! :)

Reply:
Sounds good!